components/camel-atom/pom.xml

```diff
@@ -39,6 +39,9 @@
         <camel.osgi.import>
             *
         </camel.osgi.import>
+        <camel.osgi.private>
+            com.apptasticsoftware.rssreader*
+        </camel.osgi.private>
```

Comment:
I would rather prefer that you use the wrap protocol instead especially for licensing issues

Reply:
Ok, done with wrap protocol